xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -730,7 +738,7 @@ private boolean isRequestedListener(Listener listener) {
         && hasMatchingFilter(listener.getFilterChainsList());
   }
 
-  private boolean isAddressMatching(Address address) {
+  private boolean isAddressMatching(io.envoyproxy.envoy.config.core.v3.Address address) {
```

Comment:
Can we change the import so we don't need the v3 fully qualified name?

Reply:
Done.